README.md

```diff
@@ -202,6 +202,7 @@ You can specify the browser and Node.js versions by queries (case insensitive):
 * `cover 99.5%`: most popular browsers that provide coverage.
 * `cover 99.5% in US`: same as above, with [two-letter country code].
 * `cover 99.5% in my stats`: uses [custom usage data].
+* `feature rtcpeerconnection`: browsers with support for specific features.
```

Comment:
The new query do not follow “simple English sentence” look of other queries. `supports X` can be better.

Reply:
Also we need to clarify how to get feature name

Reply:
> The new query do not follow “simple English sentence” look of other queries. `supports X` can be better.

Didn't know about project was using the "simple English sentence" concept, fixed.


> Also we need to clarify how to get feature name

Done, I've put a link similar to the regions one.